Patches/Vanilla Factions Expanded - Settlers/Scenarios/Scenarios_Classic.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+  <Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>Vanilla Factions Expanded - Settlers</li>
+    </mods>
+
+    <match Class="PatchOperationSequence">
+      <operations>
+        <!-- === Dynamine Count === -->
+        <li Class="PatchOperationReplace">
+          <xpath>/Defs/ScenarioDef[defName="VFES_Bandits"]/scenario/parts/li[10]</xpath>
```

Comment:
Is there a way to do this without resorting to using the `li`'s index? Anything changes the order and the patch might blow up.

Reply:
We're on the same boat, i tried a couple of options but this was the only one that actually worked, just told myself that if it breaks i'll find what's breaking it and fixing it.

Reply:
```suggestion
          <xpath>/Defs/ScenarioDef[defName="VFES_Bandits"]/scenario/parts/li[thingDef="VFES_Weapon_GrenadeDynamite"]</xpath>
```

How about this one? ^

Reply:
Done